woof/snake.html

```diff
@@ -0,0 +1,60 @@
+<h1><a href="./index.html"><span class="glyphicon glyphicon-arrow-left"></span></a> Rocket Race</h1>
+<div class="container">
+	<div class="block">
+
+		<p> Use the arrow keys to move the snake around to eat food! Be careful, if you eat your tail, the game ends.  </p>
+		<br><a target="_blank" href="./code/snake-code.html">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p> 1) Create a “head” to control. Make the head move at a constant rate, and make it turn left and right when the left and right arrow keys are pressed. </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=tezade">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p> 2) Create “food” for your snake to eat. Have the food be created at a constant rate and appear at random points on the screen.  </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=vimoyo/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>3) Create a scoreboard. Whenever the snake touches a food sprite, have the food sprite disappear, and make the score increase. </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=sajiki/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>4) Make it so that each time the snake eats food, the snake’s tail grows longer. Have the snake’s tail trail behind the snake.  </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=vefakoh/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>5) Create obstacles for your snake to avoid. </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=hapivod/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>6) Have the game stop if the snake touches its tail, or if it touches any of the obstacles. </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=sacuyi/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>7) When the game stops, display to the player that the game is over. </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=rosire/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>8) Make it so that when the game ends, if you press a button the game resets (position, score, and tail).  </p>
+		<br><a target="_blank" href="https://stevekrouse.github.io/embed-jsbin/?id=bofogoq/">Play this demo in full screen.</a>
+	</div>
+	<div class="block">
+
+		<p>9) At the start of the game, display how to play the game to the player. Make the snake not move to give the player a chance to take the information in. Have the snake/game start by pressing a button, and have the displayed information stop when the game starts. </p>
```

Comment:
I'd removed the sentence "Make the snake not move to give the player a chance to take the information in." It's too much information and a confusing sentiment.